### ISSUE TITLE
Fresh Server Install Trouble

### ISSUE THREAD

**Issue by CGLee65** [traccar/traccar#3979]
Hello everyone, I have just installed the traccar server on our VPS server and now I cannot access the program.  All i can get is either the file list or a 404 not found message.  I'm hoping someone can help me.

**Comment by tananaev** [comment]
Obviously it means that you are doing something wrong, but without any information no-one would be able to help you.

**Comment by CGLee65** [comment]
I realize this, What all information do you need.  We are installing this on our VPS server. When I try to run it all I get is the file list or the xml information.

Sent from Mail for Windows 10

**Comment by tananaev** [comment]
If you want some help I suggest you provide all information that can be even remotely relevant.

**Comment by CGLee65** [comment]
Ok, I finally got the login page to come up.  I registered the user and now it just keeps restarting with the following error.  HTTP 401 Unauthorized - WebApplicationException (SecurityRequestFilter:108 < ...)

Sent from Mail for Windows 10

**Comment by CGLee65** [comment]
I am also getting these.

01:09:52.586 Error in event handler for (unknown): TypeError: Cannot read property 'className' of null
    at <URL>
01:09:54.930 ext-all.js:22 GET https://www.pertdispatch.com/CAD/traccar/api/geofences?_dc=1531987794824&page=1&start=0&limit=25 401 ()
start @ ext-all.js:22
request @ ext-all.js:22
sendRequest @ ext-all.js:22
doRequest @ ext-all.js:22
read @ ext-all.js:22
doExecute @ ext-all.js:22
execute @ ext-all.js:22
flushLoad @ ext-all.js:22
(anonymous) @ ext-all.js:22
(anonymous) @ ext-all.js:22
setTimeout (async)
Ext.Function.Ext.asap @ ext-all.js:22
load @ ext-all.js:22
loadApp @ app.min.js:442
onSessionReturn @ app.min.js:442
callback @ ext-all.js:22
onComplete @ ext-all.js:22
onStateChange @ ext-all.js:22
(anonymous) @ ext-all.js:22
XMLHttpRequest.send (async)
start @ ext-all.js:22
request @ ext-all.js:22
onServerReturn @ app.min.js:441
callback @ ext-all.js:22
onComplete @ ext-all.js:22
onStateChange @ ext-all.js:22
(anonymous) @ ext-all.js:22
XMLHttpRequest.send (async)
start @ ext-all.js:22
request @ ext-all.js:22
onLaunch @ app.min.js:440
onBeforeLaunch @ ext-all.js:22
onBeforeLaunch @ ext-all.js:22
onProfilesReady @ ext-all.js:22
constructor @ ext-all.js:22
g @ ext-all.js:22
(anonymous) @ ext-all.js:22
invoke @ ext-all.js:22
on @ ext-all.js:22
Ext.onReady @ ext-all.js:22
b @ ext-all.js:22
(anonymous) @ ext-all.js:22
processCreate @ ext-all.js:22
(anonymous) @ ext-all.js:22
onBeforeCreated @ ext-all.js:22
(anonymous) @ ext-all.js:22
load @ ext-all.js:22
require @ ext-all.js:22
(anonymous) @ ext-all.js:22
a.onBeforeCreated @ ext-all.js:22
doProcess @ ext-all.js:22
process @ ext-all.js:22
d @ ext-all.js:22
create @ ext-all.js:22
define @ ext-all.js:22
Ext.application @ ext-all.js:22
(anonymous) @ app.min.js:456
01:09:54.935 ext-all.js:22 GET https://www.pertdispatch.com/CAD/traccar/api/groups?_dc=1531987794820&page=1&start=0&limit=25 401 ()
start @ ext-all.js:22
request @ ext-all.js:22
sendRequest @ ext-all.js:22
doRequest @ ext-all.js:22
read @ ext-all.js:22
doExecute @ ext-all.js:22
execute @ ext-all.js:22
flushLoad @ ext-all.js:22
(anonymous) @ ext-all.js:22
(anonymous) @ ext-all.js:22
setTimeout (async)
Ext.Function.Ext.asap @ ext-all.js:22
load @ ext-all.js:22
loadApp @ app.min.js:442
onSessionReturn @ app.min.js:442
callback @ ext-all.js:22
onComplete @ ext-all.js:22
onStateChange @ ext-all.js:22
(anonymous) @ ext-all.js:22
XMLHttpRequest.send (async)
start @ ext-all.js:22
request @ ext-all.js:22
onServerReturn @ app.min.js:441
callback @ ext-all.js:22
onComplete @ ext-all.js:22
onStateChange @ ext-all.js:22
(anonymous) @ ext-all.js:22
XMLHttpRequest.send (async)
start @ ext-all.js:22
request @ ext-all.js:22
onLaunch @ app.min.js:440
onBeforeLaunch @ ext-all.js:22
onBeforeLaunch @ ext-all.js:22
onProfilesReady @ ext-all.js:22
constructor @ ext-all.js:22
g @ ext-all.js:22
(anonymous) @ ext-all.js:22
invoke @ ext-all.js:22
on @ ext-all.js:22
Ext.onReady @ ext-all.js:22
b @ ext-all.js:22
(anonymous) @ ext-all.js:22
processCreate @ ext-all.js:22
(anonymous) @ ext-all.js:22
onBeforeCreated @ ext-all.js:22
(anonymous) @ ext-all.js:22
load @ ext-all.js:22
require @ ext-all.js:22
(anonymous) @ ext-all.js:22
a.onBeforeCreated @ ext-all.js:22
doProcess @ ext-all.js:22
process @ ext-all.js:22
d @ ext-all.js:22
create @ ext-all.js:22
define @ ext-all.js:22
Ext.application @ ext-all.js:22
(anonymous) @ app.min.js:456
01:09:54.936 ext-all.js:22 GET https://www.pertdispatch.com/CAD/traccar/api/drivers?_dc=1531987794823&page=1&start=0&limit=25 401 ()
start @ ext-all.js:22
request @ ext-all.js:22
sendRequest @ ext-all.js:22
doRequest @ ext-all.js:22
read @ ext-all.js:22
doExecute @ ext-all.js:22
execute @ ext-all.js:22
flushLoad @ ext-all.js:22
(anonymous) @ ext-all.js:22
(anonymous) @ ext-all.js:22
setTimeout (async)
Ext.Function.Ext.asap @ ext-all.js:22
load @ ext-all.js:22
loadApp @ app.min.js:442
onSessionReturn @ app.min.js:442
callback @ ext-all.js:22
onComplete @ ext-all.js:22
onStateChange @ ext-all.js:22
(anonymous) @ ext-all.js:22
XMLHttpRequest.send (async)
start @ ext-all.js:22
request @ ext-all.js:22
onServerReturn @ app.min.js:441
callback @ ext-all.js:22
onComplete @ ext-all.js:22
onStateChange @ ext-all.js:22
(anonymous) @ ext-all.js:22
XMLHttpRequest.send (async)
start @ ext-all.js:22
request @ ext-all.js:22
onLaunch @ app.min.js:440
onBeforeLaunch @ ext-all.js:22
onBeforeLaunch @ ext-all.js:22
onProfilesReady @ ext-all.js:22
constructor @ ext-all.js:22
g @ ext-all.js:22
(anonymous) @ ext-all.js:22
invoke @ ext-all.js:22
on @ ext-all.js:22
Ext.onReady @ ext-all.js:22
b @ ext-all.js:22
(anonymous) @ ext-all.js:22
processCreate @ ext-all.js:22
(anonymous) @ ext-all.js:22
onBeforeCreated @ ext-all.js:22
(anonymous) @ ext-all.js:22
load @ ext-all.js:22
require @ ext-all.js:22
(anonymous) @ ext-all.js:22
a.onBeforeCreated @ ext-all.js:22
doProcess @ ext-all.js:22
process @ ext-all.js:22
d @ ext-all.js:22
create @ ext-all.js:22
define @ ext-all.js:22
Ext.application @ ext-all.js:22
(anonymous) @ app.min.js:456
01:09:54.938 ext-all.js:22 GET https://www.pertdispatch.com/CAD/traccar/api/maintenances?_dc=1531987794833&page=1&start=0&limit=25 401 ()
start @ ext-all.js:22
request @ ext-all.js:22
sendRequest @ ext-all.js:22
doRequest @ ext-all.js:22
read @ ext-all.js:22
doExecute @ ext-all.js:22
execute @ ext-all.js:22
flushLoad @ ext-all.js:22
(anonymous) @ ext-all.js:22
(anonymous) @ ext-all.js:22
setTimeout (async)
Ext.Function.Ext.asap @ ext-all.js:22
load @ ext-all.js:22
loadApp @ app.min.js:442
onSessionReturn @ app.min.js:442
callback @ ext-all.js:22
onComplete @ ext-all.js:22
onStateChange @ ext-all.js:22
(anonymous) @ ext-all.js:22
XMLHttpRequest.send (async)
start @ ext-all.js:22
request @ ext-all.js:22
onServerReturn @ app.min.js:441
callback @ ext-all.js:22
onComplete @ ext-all.js:22
onStateChange @ ext-all.js:22
(anonymous) @ ext-all.js:22
XMLHttpRequest.send (async)
start @ ext-all.js:22
request @ ext-all.js:22
onLaunch @ app.min.js:440
onBeforeLaunch @ ext-all.js:22
onBeforeLaunch @ ext-all.js:22
onProfilesReady @ ext-all.js:22
constructor @ ext-all.js:22
g @ ext-all.js:22
(anonymous) @ ext-all.js:22
invoke @ ext-all.js:22
on @ ext-all.js:22
Ext.onReady @ ext-all.js:22
b @ ext-all.js:22
(anonymous) @ ext-all.js:22
processCreate @ ext-all.js:22
(anonymous) @ ext-all.js:22
onBeforeCreated @ ext-all.js:22
(anonymous) @ ext-all.js:22
load @ ext-all.js:22
require @ ext-all.js:22
(anonymous) @ ext-all.js:22
a.onBeforeCreated @ ext-all.js:22
doProcess @ ext-all.js:22
process @ ext-all.js:22
d @ ext-all.js:22
create @ ext-all.js:22
define @ ext-all.js:22
Ext.application @ ext-all.js:22
(anonymous) @ app.min.js:456
01:09:54.939 ext-all.js:22 GET https://www.pertdispatch.com/CAD/traccar/api/calendars?_dc=1531987794831&page=1&start=0&limit=25 401 ()
start @ ext-all.js:22
request @ ext-all.js:22
sendRequest @ ext-all.js:22
doRequest @ ext-all.js:22
read @ ext-all.js:22
doExecute @ ext-all.js:22
execute @ ext-all.js:22
flushLoad @ ext-all.js:22
(anonymous) @ ext-all.js:22
(anonymous) @ ext-all.js:22
setTimeout (async)
Ext.Function.Ext.asap @ ext-all.js:22
load @ ext-all.js:22
loadApp @ app.min.js:442
onSessionReturn @ app.min.js:442
callback @ ext-all.js:22
onComplete @ ext-all.js:22
onStateChange @ ext-all.js:22
(anonymous) @ ext-all.js:22
XMLHttpRequest.send (async)
start @ ext-all.js:22
request @ ext-all.js:22
onServerReturn @ app.min.js:441
callback @ ext-all.js:22
onComplete @ ext-all.js:22
onStateChange @ ext-all.js:22
(anonymous) @ ext-all.js:22
XMLHttpRequest.send (async)
start @ ext-all.js:22
request @ ext-all.js:22
onLaunch @ app.min.js:440
onBeforeLaunch @ ext-all.js:22
onBeforeLaunch @ ext-all.js:22
onProfilesReady @ ext-all.js:22
constructor @ ext-all.js:22
g @ ext-all.js:22
(anonymous) @ ext-all.js:22
invoke @ ext-all.js:22
on @ ext-all.js:22
Ext.onReady @ ext-all.js:22
b @ ext-all.js:22
(anonymous) @ ext-all.js:22
processCreate @ ext-all.js:22
(anonymous) @ ext-all.js:22
onBeforeCreated @ ext-all.js:22
(anonymous) @ ext-all.js:22
load @ ext-all.js:22
require @ ext-all.js:22
(anonymous) @ ext-all.js:22
a.onBeforeCreated @ ext-all.js:22
doProcess @ ext-all.js:22
process @ ext-all.js:22
d @ ext-all.js:22
create @ ext-all.js:22
define @ ext-all.js:22
Ext.application @ ext-all.js:22
(anonymous) @ app.min.js:456
01:09:54.940 ext-all.js:22 GET https://www.pertdispatch.com/CAD/traccar/api/commands/types?_dc=1531987794837&page=1&start=0&limit=25 401 ()
start @ ext-all.js:22
request @ ext-all.js:22
sendRequest @ ext-all.js:22
doRequest @ ext-all.js:22
read @ ext-all.js:22
doExecute @ ext-all.js:22
execute @ ext-all.js:22
flushLoad @ ext-all.js:22
(anonymous) @ ext-all.js:22
(anonymous) @ ext-all.js:22
setTimeout (async)
Ext.Function.Ext.asap @ ext-all.js:22
load @ ext-all.js:22
loadApp @ app.min.js:442
onSessionReturn @ app.min.js:442
callback @ ext-all.js:22
onComplete @ ext-all.js:22
onStateChange @ ext-all.js:22
(anonymous) @ ext-all.js:22
XMLHttpRequest.send (async)
start @ ext-all.js:22
request @ ext-all.js:22
onServerReturn @ app.min.js:441
callback @ ext-all.js:22
onComplete @ ext-all.js:22
onStateChange @ ext-all.js:22
(anonymous) @ ext-all.js:22
XMLHttpRequest.send (async)
start @ ext-all.js:22
request @ ext-all.js:22
onLaunch @ app.min.js:440
onBeforeLaunch @ ext-all.js:22
onBeforeLaunch @ ext-all.js:22
onProfilesReady @ ext-all.js:22
constructor @ ext-all.js:22
g @ ext-all.js:22
(anonymous) @ ext-all.js:22
invoke @ ext-all.js:22
on @ ext-all.js:22
Ext.onReady @ ext-all.js:22
b @ ext-all.js:22
(anonymous) @ ext-all.js:22
processCreate @ ext-all.js:22
(anonymous) @ ext-all.js:22
onBeforeCreated @ ext-all.js:22
(anonymous) @ ext-all.js:22
load @ ext-all.js:22
require @ ext-all.js:22
(anonymous) @ ext-all.js:22
a.onBeforeCreated @ ext-all.js:22
doProcess @ ext-all.js:22
process @ ext-all.js:22
d @ ext-all.js:22
create @ ext-all.js:22
define @ ext-all.js:22
Ext.application @ ext-all.js:22
(anonymous) @ app.min.js:456
01:09:55.023 ext-all.js:22 GET https://www.pertdispatch.com/CAD/traccar/api/attributes/computed?_dc=1531987794835&page=1&start=0&limit=25 401 ()
start @ ext-all.js:22
request @ ext-all.js:22
sendRequest @ ext-all.js:22
doRequest @ ext-all.js:22
read @ ext-all.js:22
doExecute @ ext-all.js:22
execute @ ext-all.js:22
flushLoad @ ext-all.js:22
(anonymous) @ ext-all.js:22
(anonymous) @ ext-all.js:22
setTimeout (async)
Ext.Function.Ext.asap @ ext-all.js:22
load @ ext-all.js:22
loadApp @ app.min.js:442
onSessionReturn @ app.min.js:442
callback @ ext-all.js:22
onComplete @ ext-all.js:22
onStateChange @ ext-all.js:22
(anonymous) @ ext-all.js:22
XMLHttpRequest.send (async)
start @ ext-all.js:22
request @ ext-all.js:22
onServerReturn @ app.min.js:441
callback @ ext-all.js:22
onComplete @ ext-all.js:22
onStateChange @ ext-all.js:22
(anonymous) @ ext-all.js:22
XMLHttpRequest.send (async)
start @ ext-all.js:22
request @ ext-all.js:22
onLaunch @ app.min.js:440
onBeforeLaunch @ ext-all.js:22
onBeforeLaunch @ ext-all.js:22
onProfilesReady @ ext-all.js:22
constructor @ ext-all.js:22
g @ ext-all.js:22
(anonymous) @ ext-all.js:22
invoke @ ext-all.js:22
on @ ext-all.js:22
Ext.onReady @ ext-all.js:22
b @ ext-all.js:22
(anonymous) @ ext-all.js:22
processCreate @ ext-all.js:22
(anonymous) @ ext-all.js:22
onBeforeCreated @ ext-all.js:22
(anonymous) @ ext-all.js:22
load @ ext-all.js:22
require @ ext-all.js:22
(anonymous) @ ext-all.js:22
a.onBeforeCreated @ ext-all.js:22
doProcess @ ext-all.js:22
process @ ext-all.js:22
d @ ext-all.js:22
create @ ext-all.js:22
define @ ext-all.js:22
Ext.application @ ext-all.js:22
(anonymous) @ app.min.js:456
01:09:55.024 ext-all.js:22 GET https://www.pertdispatch.com/CAD/traccar/api/notifications/types?_dc=1531987794840&page=1&start=0&limit=25 401 ()
start @ ext-all.js:22
request @ ext-all.js:22
sendRequest @ ext-all.js:22
doRequest @ ext-all.js:22
read @ ext-all.js:22
doExecute @ ext-all.js:22
execute @ ext-all.js:22
flushLoad @ ext-all.js:22
(anonymous) @ ext-all.js:22
(anonymous) @ ext-all.js:22
setTimeout (async)
Ext.Function.Ext.asap @ ext-all.js:22
load @ ext-all.js:22
loadApp @ app.min.js:442
onSessionReturn @ app.min.js:442
callback @ ext-all.js:22
onComplete @ ext-all.js:22
onStateChange @ ext-all.js:22
(anonymous) @ ext-all.js:22
XMLHttpRequest.send (async)
start @ ext-all.js:22
request @ ext-all.js:22
onServerReturn @ app.min.js:441
callback @ ext-all.js:22
onComplete @ ext-all.js:22
onStateChange @ ext-all.js:22
(anonymous) @ ext-all.js:22
XMLHttpRequest.send (async)
start @ ext-all.js:22
request @ ext-all.js:22
onLaunch @ app.min.js:440
onBeforeLaunch @ ext-all.js:22
onBeforeLaunch @ ext-all.js:22
onProfilesReady @ ext-all.js:22
constructor @ ext-all.js:22
g @ ext-all.js:22
(anonymous) @ ext-all.js:22
invoke @ ext-all.js:22
on @ ext-all.js:22
Ext.onReady @ ext-all.js:22
b @ ext-all.js:22
(anonymous) @ ext-all.js:22
processCreate @ ext-all.js:22
(anonymous) @ ext-all.js:22
onBeforeCreated @ ext-all.js:22
(anonymous) @ ext-all.js:22
load @ ext-all.js:22
require @ ext-all.js:22
(anonymous) @ ext-all.js:22
a.onBeforeCreated @ ext-all.js:22
doProcess @ ext-all.js:22
process @ ext-all.js:22
d @ ext-all.js:22
create @ ext-all.js:22
define @ ext-all.js:22
Ext.application @ ext-all.js:22
(anonymous) @ app.min.js:456
01:09:55.039 ext-all.js:22 GET https://www.pertdispatch.com/CAD/traccar/api/commands?_dc=1531987794838&page=1&start=0&limit=25 401 ()
start @ ext-all.js:22
request @ ext-all.js:22
sendRequest @ ext-all.js:22
doRequest @ ext-all.js:22
read @ ext-all.js:22
doExecute @ ext-all.js:22
execute @ ext-all.js:22
flushLoad @ ext-all.js:22
(anonymous) @ ext-all.js:22
(anonymous) @ ext-all.js:22
setTimeout (async)
Ext.Function.Ext.asap @ ext-all.js:22
load @ ext-all.js:22
loadApp @ app.min.js:442
onSessionReturn @ app.min.js:442
callback @ ext-all.js:22
onComplete @ ext-all.js:22
onStateChange @ ext-all.js:22
(anonymous) @ ext-all.js:22
XMLHttpRequest.send (async)
start @ ext-all.js:22
request @ ext-all.js:22
onServerReturn @ app.min.js:441
callback @ ext-all.js:22
onComplete @ ext-all.js:22
onStateChange @ ext-all.js:22
(anonymous) @ ext-all.js:22
XMLHttpRequest.send (async)
start @ ext-all.js:22
request @ ext-all.js:22
onLaunch @ app.min.js:440
onBeforeLaunch @ ext-all.js:22
onBeforeLaunch @ ext-all.js:22
onProfilesReady @ ext-all.js:22
constructor @ ext-all.js:22
g @ ext-all.js:22
(anonymous) @ ext-all.js:22
invoke @ ext-all.js:22
on @ ext-all.js:22
Ext.onReady @ ext-all.js:22
b @ ext-all.js:22
(anonymous) @ ext-all.js:22
processCreate @ ext-all.js:22
(anonymous) @ ext-all.js:22
onBeforeCreated @ ext-all.js:22
(anonymous) @ ext-all.js:22
load @ ext-all.js:22
require @ ext-all.js:22
(anonymous) @ ext-all.js:22
a.onBeforeCreated @ ext-all.js:22
doProcess @ ext-all.js:22
process @ ext-all.js:22
d @ ext-all.js:22
create @ ext-all.js:22
define @ ext-all.js:22
Ext.application @ ext-all.js:22
(anonymous) @ app.min.js:456
01:09:55.040 ext-all.js:22 GET https://www.pertdispatch.com/CAD/traccar/api/notifications?_dc=1531987794842&page=1&start=0&limit=25 401 ()
start @ ext-all.js:22
request @ ext-all.js:22
sendRequest @ ext-all.js:22
doRequest @ ext-all.js:22
read @ ext-all.js:22
doExecute @ ext-all.js:22
execute @ ext-all.js:22
flushLoad @ ext-all.js:22
(anonymous) @ ext-all.js:22
(anonymous) @ ext-all.js:22
setTimeout (async)
Ext.Function.Ext.asap @ ext-all.js:22
load @ ext-all.js:22
loadApp @ app.min.js:443
onSessionReturn @ app.min.js:442
callback @ ext-all.js:22
onComplete @ ext-all.js:22
onStateChange @ ext-all.js:22
(anonymous) @ ext-all.js:22
XMLHttpRequest.send (async)
start @ ext-all.js:22
request @ ext-all.js:22
onServerReturn @ app.min.js:441
callback @ ext-all.js:22
onComplete @ ext-all.js:22
onStateChange @ ext-all.js:22
(anonymous) @ ext-all.js:22
XMLHttpRequest.send (async)
start @ ext-all.js:22
request @ ext-all.js:22
onLaunch @ app.min.js:440
onBeforeLaunch @ ext-all.js:22
onBeforeLaunch @ ext-all.js:22
onProfilesReady @ ext-all.js:22
constructor @ ext-all.js:22
g @ ext-all.js:22
(anonymous) @ ext-all.js:22
invoke @ ext-all.js:22
on @ ext-all.js:22
Ext.onReady @ ext-all.js:22
b @ ext-all.js:22
(anonymous) @ ext-all.js:22
processCreate @ ext-all.js:22
(anonymous) @ ext-all.js:22
onBeforeCreated @ ext-all.js:22
(anonymous) @ ext-all.js:22
load @ ext-all.js:22
require @ ext-all.js:22
(anonymous) @ ext-all.js:22
a.onBeforeCreated @ ext-all.js:22
doProcess @ ext-all.js:22
process @ ext-all.js:22
d @ ext-all.js:22
create @ ext-all.js:22
define @ ext-all.js:22
Ext.application @ ext-all.js:22
(anonymous) @ app.min.js:456
01:09:55.056 ext-all.js:22 GET https://www.pertdispatch.com/CAD/traccar/api/devices?_dc=1531987794866&page=1&start=0&limit=25 401 ()

Sent from Mail for Windows 10

**Comment by tananaev** [comment]
I removed your emailed comments because they were just unreadable. Please answer through github and properly format your messages.

By the way, from your last comment it seems like you haven't actually installed Traccar properly.

**Comment by CGLee65** [comment]
Ok, I was listing the errors.  The main error I get now when trying to log in is the following. HTTP 401 Unauthorized - WebApplicationException (SecurityRequestFilter:108 < ...)

Sent from Mail for Windows 10

**Comment by CGLee65** [comment]
Ok, what am I suppose to post.  You asked for the information and I am providing it

**Comment by CGLee65** [comment]
I stated that now when I try to login it just keeps looping and I tried to provide the error it was giving me

**Comment by tananaev** [comment]
It seems like you ignore most of what I'm writing. I would recommend you take your time and carefully read what I wrote and think about it.

**Comment by CGLee65** [comment]
It comes up but we keep getting an error and just restarts.  I can't provide the errors here cause they get deleted.

**Comment by tananaev** [comment]
Which instructions? That's kind of information I want to know. I'll give you last chance to take some time and carefully write down everything that you did from the start. Properly write a github message with readable formatting.

**Comment by CGLee65** [comment]
I don’t know why the attitude with us, we are only trying to get some help with this.  We have followed the Install on VPS instructions on the traccar.org website for the install on our VPS server.  The service started correctly and the login page comes up as it is suppose to.  We then registered our dispatch email address, and it was successfully registered.  Once we try to login to traccar on our URL it loops and comes up with the error message I tried to include last time.

Sent from Mail for Windows 10

**Comment by tananaev** [comment]
OK, can you please explain how, by following single "Install on VPS" guide, you managed to install your server under `/CAD/traccar/` path on your server.

**Comment by CGLee65** [comment]
We created the my php database as traccar, we then downloaded and uploaded the traccar-Linux.zip to our server. Then unzipped the file to the folder public_html/CAD/traccar/ location, then our host ran the install from the terminal and set our config to point to the database after I edited the traccar.xml file with the DB user name and password.  At that point the host finished the config file and we where able to open it at https://www.pertdispatch.com/CAD/traccar/ at which we get the login.  I registered the main dispatch email which it said it had registered correctly. I then tried to login to the traccar with that email and password and that is when it started to loop from the map view to the error box and just keep doing that.

Sent from Mail for Windows 10
